### PR TITLE
Created new pack for Trend Micro TippingPoint

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5085

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/TrendMicroTippingPoint/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Trend Micro TippingPoint
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure the Security Management System (SMS) of Trend Micro TippingPoint to forward Syslog messages in CEF format.
+
+### Log in to the TippingPoint system and follow the below steps:
+1. On the **Admin Navigation** menu, select **Server Properties**.
+2. Select the **Syslog** tab.
+3. Under "Remote Syslog for Events" Click **New**.
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroTippingPoint/README.md:9]:
```suggestion
3. Under **Remote Syslog for Events**, click **New**.
```

File: Packs/TrendMicroTippingPoint/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+4. Select the **Enable** check box.
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroTippingPoint/README.md:10]:
```suggestion
4. Select the **Enable** checkbox.
```

File: Packs/TrendMicroTippingPoint/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+5. Configure the following values:
+   - Syslog Server - The IP address of the XSIAM Broker.
+   - Port - Type 514 as the port address.
+   - Log Type - Select ArcSight CEF Format v4.2.
+   - Facility - Select Log Audit.
+   - Severity - Select Severity in Event.
+   - Delimiter - Select Space as the delimiter for the generated logs.
+   - Include Timestamp in Header - Select Use original event timestamp.
+   - Select the Include SMS Hostname in Header check box.
+6. Click OK.
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroTippingPoint/README.md:20]:
```suggestion
6. Click **OK**.
```

File: Packs/TrendMicroTippingPoint/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+In order to use the collector, use the [Broker VM](#broker-vm) option.
+
+### Broker VM
+To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
+
+You can configure the specific vendor and product for this instance.
+
+
+1. Navigate to **Settings** > **Configuration** > **Data Broker** > **Broker VMs**. 
+2. Right-click, and select **Syslog Collector** > **Configure**.
```
Comment (ShirleyDenkberg) [inline Packs/TrendMicroTippingPoint/README.md:36]: Are there additional steps that are missing?

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

File: Packs/TrendMicroTippingPoint/README.md (hunk 4 above)
Comment (yasta5) [reply Packs/TrendMicroTippingPoint/README.md:36]: No.

**Review by evisochek — APPROVED** [review]

**Pull request merged**